<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDiskEncryptConfig</class>
 <widget class="QDialog" name="UDiskEncryptConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密钥配置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>加密算法</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbNoEnc">
        <property name="text">
         <string>不加密</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAES">
        <property name="text">
         <string>AES256</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSM4">
        <property name="text">
         <string>SM4</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>密钥：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePassword">
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbNoEnc</sender>
   <signal>toggled(bool)</signal>
   <receiver>UDiskEncryptConfig</receiver>
   <slot>sltNoEnc(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSM4</sender>
   <signal>toggled(bool)</signal>
   <receiver>UDiskEncryptConfig</receiver>
   <slot>sltSM4(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbConfirm</sender>
   <signal>clicked()</signal>
   <receiver>UDiskEncryptConfig</receiver>
   <slot>sltConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbAES</sender>
   <signal>clicked(bool)</signal>
   <receiver>UDiskEncryptConfig</receiver>
   <slot>sltAES(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sltNoEnc(bool)</slot>
  <slot>sltSM4(bool)</slot>
  <slot>sltConfirm()</slot>
  <slot>sltAES(bool)</slot>
 </slots>
</ui>
